docs/api.md

```diff
@@ -0,0 +1,376 @@
+# API
+
+## Surface
+
+Container for all other ART components
+
+|   Prop   |     Type     | Default |
+| :------: | :----------: | :-----: |
+|  height  |   `number`   |   `0`   |
+|  width   |   `number`   |   `0`   |
+|  style   |    `any`     |   ---   |
+| children | `React.Node` |   ---   |
+
+```js
+import {Dimensions} from 'react-native';
+import {Surface} from '@react-native-community/art';
+
+const SURFACE_DIMENSIONS = Dimensions.get('window').width;
+
+function Heart() {
+  return (
+    <Surface width={SURFACE_DIMENSIONS} height={SURFACE_DIMENSIONS}>
+      {renderARTShapes()}
+    </Surface>
+  );
+}
+```
+
+## Group
+
+Container to combine shapes or other groups into hierarchies that can be transformed as a set.
+
+|       Prop        |       Type       | Default |
+| :---------------: | :--------------: | :-----: |
+|  ...opacityProps  |  `OpacityProps`  |   ---   |
+| ...transformProps | `TransformProps` |   ---   |
+|     children      |   `React.Node`   |   ---   |
+
+```js
+import {Dimensions} from 'react-native';
+import {Surface, Group} from '@react-native-community/art';
+
+const SURFACE_DIMENSIONS = Dimensions.get('window').width;
+
+function GrouppedHearts() {
+  return (
+    <Surface width={SURFACE_DIMENSIONS} height={SURFACE_DIMENSIONS}>
+      <Group>{renderARTGroupContents()}</Group>
+    </Surface>
+  );
+}
+```
+
+## Shape
+
+Used to draw arbitrary vector shapes from Path. Shape implements Transform as a mixin which means it has all transform methods available for moving, scaling and rotating a shape.
+
+|       Prop        |              Type               |  Default  |
+| :---------------: | :-----------------------------: | :-------: |
+|  ...opacityProps  |         `OpacityProps`          |    ---    |
+| ...transformProps |        `TransformProps`         |    ---    |
+|       fill        |        `string \| Brush`        |    ---    |
+|      stroke       |            `string`             |    ---    |
+|     strokeCap     | `'butt' \| 'square' \| 'round'` | `'round'` |
+|    strokeDash     |         `Array<number>`         |    ---    |
+|    strokeJoin     | `'miter' \| 'bevel' \| 'round'` | `'round'` |
+|    strokeWidth    |            `number`             |    `1`    |
+|         d         |        `string \| Path`         |    ---    |
+|       width       |            `number`             |    `0`    |
+|      height       |            `number`             |    `0`    |
+|     children      |          `React.Node`           |    ---    |
+
+```js
+import {Dimensions} from 'react-native';
+import {Surface, Shape} from '@react-native-community/art';
+
+const SURFACE_DIMENSIONS = Dimensions.get('window').width;
+
+function Heart() {
+  return (
+    <Surface width={SURFACE_DIMENSIONS} height={SURFACE_DIMENSIONS}>
```

Comment:
thinking about it more, I think it's gonna be more straight-forward to provide some fixed dimensions and not use Dimensions, it's an extra cognitive load.

Reply:
Good idea. Will replace it with some fixed values.